Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -900,6 +901,13 @@ extension ChannelPipeline {
         try sync.addHandler(handler)
     }
 
+    func syncAddSOCKSProxyHandler(host: String, port: Int) throws {
+        let address = try SocketAddress.makeAddressResolvingHost(host, port: port)
+        let handler = SOCKSClientHandler(targetAddress: .address(address))
```

Comment:
I think you resolved my feedback here but this is still valid: I think we should be using the hostname form here.

Reply:
Oh I misunderstood - I switched to using `let address = try SocketAddress.makeAddressResolvingHost(host, port: port)`